WordPress/WordPressShareExtension/MainShareViewController.swift

```diff
@@ -96,14 +96,8 @@ private extension MainShareViewController {
 
         let shareNavController = UINavigationController(rootViewController: editorController)
 
-        if #available(iOS 13, *), editorController.originatingExtension == .saveToDraft {
-            // iOS 13 has proper animations and presentations for share and action sheets. So the `else` case should be removed when iOS 13 is minimum.
-            // We  need to make sure we don't end up with stacked modal view controllers by using this:
-            shareNavController.modalPresentationStyle = .overFullScreen
-        } else {
-            shareNavController.transitioningDelegate = extensionTransitioningManager
-            shareNavController.modalPresentationStyle = .custom
-        }
+        // We need to make sure we don't end up with stacked modal view controllers by using this:
+        shareNavController.modalPresentationStyle = .overFullScreen
```

Comment:
This is not an equivalent conversion, as you lost the `if editorController.originatingExtension == .saveToDraft` condition.

Reply:
Great catch! Thanks!

Reply:
@AliSoftware I addressed this in [`6d49db7` (#19513)](https://github.com/wordpress-mobile/WordPress-iOS/pull/19513/commits/6d49db787d1d52b2ed0643b02a21e66e030eb73e), thanks again.

 I started looking at how to add a unit test to catch
this issue but I run into a number of problems.

1. The code is in the WordPressShareExtension target and we cannot test
   that one directly; Xcode doesn't let you create test target against
   app extensions
2. Even if we could access the code, the logic depends on
   `editorController`, an instance property that sets the state based
   whether `Bundle.main.bundleIdentifier` contains "DraftAction". That
   is something we cannot modify at test run time.

In short, to test this trivial behavior, we'll need to put in place a
far-from trivial test harness. The idea I have at the moment would be to
extract as much of the logic shared between WordPressShareExtension,
WordPressDraftActionExtension, and their two Jetpack counter parts into
a framework, which we could then test in isolation and design in such a
way that we don't need to inspect the bundle identifier at runtime.

It's worth saying that folks were aware of the code limitations at the
time of implementing the behavior but must have been pressed for time to
ship a bug fix, as evidenced by the comment in the `editorController`
implementation.